code/game/objects/items/devices/flashlight.dm

```diff
@@ -428,3 +428,11 @@
 	item_state = "flashdark"
 	brightness_on = 2.5
 	flashlight_power = -3
+
+/obj/item/device/flashlight/eyelight
+	name = "eyelight"
+	desc = "This shouldn't exist outside of someone's head, how are you seeing this?"
+	brightness_on = 15
+	flashlight_power = 1
+	flags = CONDUCT | DROPDEL
+	actions_types = list()

```

Comment:
This is a hack, we *really* should just make organs inside the person. Not your fault.